src/components/util/Bubble.tsx

```diff
@@ -75,17 +85,32 @@ export default function Bubble(props: Props) {
 		};
 	}, [value]);
 
+	React.useImperativeHandle(ref, () => ({
+		selectionStart,
```

Comment:
remove everything but `focus`

we can add 'keyup' event listener in `useEffect[isEditing]` just below `inputRef.current?.focus()`

accept selectNext, selectPrev as optional function props